packages-ts/terra-sdk/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@chainlink/terra-sdk",
+  "version": "0.1.0",
+  "repository": "https://github.com/smartcontractkit/chainlink-terra",
+  "license": "MIT",
+  "engines": {
+    "node": ">= 14"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist/**/*",
+    "src",
+    "LICENSE",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "exports": {
+    "import": "./dist/esm/index.js",
+    "require": "./dist/cjs/index.js"
+  },
```

Comment:
@HenryNguyen5 hey hey, we could use your expertise here! 
We're trying to simply package this simple package :)

Could you help us by reviewing and suggesting the proper configs to do so? We'd love to support browsers and multiple module options, but for now, we're just looking to support the usual general case and get this over quickly.

Any suggestions?

Reply:
Solana could use some help on this too: https://github.com/smartcontractkit/chainlink-solana/pull/224

These might be of help, I compared tsconfigs for 
- [Solana](https://github.com/solana-labs/solana-program-library/tree/master/token/js)
- [Anchor](https://github.com/project-serum/anchor/tree/master/ts)
- [SaberSwap](https://github.com/saber-hq/saber-common/tree/master/packages/stableswap-sdk)

The last one seemed the cleanest but it imported a separate base tsconfig from somewhere else.

Reply:
We could offer only ESM and offer CJS later if necessary https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
I would suggest that we stick with CJS output for now. TSC support for .mjs is experimental due to mjs + cjs ecosystem fragmentation. The module mode with the most support by far is still .cjs. I would only consider outputting ESM style modules if browser support + tree shaking is a high priority. 

CJS is the most compatible module format you can target, can't go wrong with it for now. ESM is a really big headache that isn't worth supporting unless you have a very specific use case you must optimize for, like tree shaking.

See https://github.com/microsoft/TypeScript/issues/46452

Reply:
Another pain point I can think of is that ESM modules can easily import CJS files, but CJS files must asynchronously import ESM files.

If this project depends on ESM style dependencies, you might be forced to output ESM anyway, since it'll be too much friction to continue using CJS modules.

I would do an evaluation of your existing dependencies, if any of them are ESM or have major versions that have upgraded to ESM, that might be the best path forward

Reply:
Let's stick with a dual system then